projects/components/src/table/table.component.ts

```diff
@@ -517,7 +528,7 @@ export class TableComponent
   private pageDataFromUrl(params: ParamMap): Partial<PageEvent> | undefined {
     return this.syncWithUrl
       ? {
-          pageSize: new NumberCoercer().coerce(params.get(TableComponent.PAGE_SIZE_URL_PARAM)),
+          pageSize: new NumberCoercer().coerce(params.get(TableComponent.PAGE_SIZE_URL_PARAM)) ?? this.pageSize,
```

Comment:
Same effect, but slightly more readable IMO if we declare defaultValue explicitly. 
```
pageSize: new NumberCoercer({defaultvalue: this.pageSize}).coerce(params.get(TableComponent.PAGE_SIZE_URL_PARAM))
```